### ISSUE TITLE
USER ISSUE: No Local Game possible

### ISSUE THREAD

**Issue by SushySan** [StrangeLoopGames/EcoIssues#6737]
**Version:** 0.7.3.0 beta

**Steps to Reproduce:**
Create a new local Game or load existing one	

**Expected behavior:**
Loading	

**Actual behavior:**
It loads but freeze with a running Wheel no further step.

**Comment by metachronism** [comment]
Hey @SushySan, do you have a copy of your storage folder you can send us?

**Comment by SushySan** [comment]
Hihi!

I had a chat with the support on discord.
We figured out that it was a update problem within last steam patch.

We needed to deinstall the game.
Then delete manual all game folders.
Also delete manual the hidden Eco folder within appdata folder which was difficult but finally a reinstall could bring the game back to work.

BR
Sascha



Diese Nachricht wurde ihnen präsentiert von:

[  ] dem freundlichen Nachbarn von Nebenan
[  ] dem Marvel Superhelden um die Ecke
[x] dem Sascha sein iPhone
[  ] www.oderdochweranders.de
